### DISCUSSION TITLE
Help Needed: Adapting `pre_trained_policy_action.py` for Dual Policy in Multi-Policy Hierarchical RL

### DISCUSSION THREAD

**Discussion by H-Hisamichi** [isaac-sim/IsaacLab#1545] [Q&A]
Dear All,

I need help.

I’ve been working on a project involving hierarchical reinforcement learning with two low-level policies, using the navigation demo code as a reference, but I’ve hit a roadblock.

My plan is to input the two policy information into `pre_trained_policy_action.py`, process them based on commands, select the appropriate policy for each actor, generate a new action tensor, and apply it to the simulation using `self._low_level_action_term.apply_actions()`.

However, I’m struggling to correctly handle the command retrieval process within this script.

What is the proper way to implement command retrieval in `pre_trained_policy_action.py`?
Alternatively, would it be inappropriate to modify this script to include custom logic?

Any help would be greatly appreciated!

**Comment by RandomOakForest** [comment]
Thanks for posting this. The _discussion_ is closed, but let us know if you'd like to reopen it and we will aim to follow up soon.

**Comment by H-Hisamichi** [comment]
Hello @RandomOakForest,

Thank you for following up. My multi-policy hierarchical reinforcement learning project is currently on hold. I’d appreciate your support if I resume it in the future.
